canonical/lxd-ui
# Link images from storage pool used by section

## Done

- add link for images on storage pool used by section
- append project to the image names in storage pool used by section

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - ensure images are linked and displayed correctly on https://lxd-ui-605.demos.haus/ui/project/default/storage/detail/ceph-pool
